File: View Binding/Activity Main/MainActivity.java
import androidx.appcompat.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;
import android.widget.Toast;

import com.example.practice1.databinding.ActivityMainBinding;

public class MainActivity extends AppCompatActivity {

    ActivityMainBinding activityMainBinding; // just type binding. find the binding that match with you xml file

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        activityMainBinding = ActivityMainBinding.inflate(getLayoutInflater());
        //setContentView(R.layout.activity_main);
        setContentView(activityMainBinding.getRoot());

        activityMainBinding.textView.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                Toast.makeText(getApplicationContext(), "hERE", Toast.LENGTH_SHORT).show();
            }
        });
    }
}

File: View Binding/MainActivity.kt
class MainActivity : AppCompatActivity() {

    private  lateinit var  binding: ActivityMainBinding


    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        binding = ActivityMainBinding.inflate(layoutInflater)
        setContentView(binding.root)

       binding.btnClick.setOnClickListener {
           Toast.makeText(this,"You have done it ",Toast.LENGTH_LONG).show()
       }
    }
}
